timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -81,33 +97,173 @@ limitations under the License.
           <tbody
-            :class="$vuetify.theme.dark ? (expanded ? 'dark-hover dark-bg' :'dark-hover') : (expanded ? 'light-hover light-bg' : 'light-hover')"
+            :class="
+              $vuetify.theme.dark
+                ? expanded
+                  ? 'dark-hover dark-bg'
+                  : 'dark-hover'
+                : expanded
+                ? 'light-hover light-bg'
+                : 'light-hover'
+            "
           >
             <tr class="pr-3">
-              <td width="80" style="border: none">
-                <strong v-if="timeline.analysis_status == 'ERROR'">Error:</strong>
-                <strong v-else>Verdict:</strong>
+              <td width="105" style="border: none">
+                <strong>Result:</strong>
               </td>
               <td style="border: none">
                 <span>
-                  {{ timeline.verdict }}
+                  {{ verboseAnalyzerOutput.result_summary }}
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td style="border: none">
+                <strong>Priority:</strong>
+              </td>
+              <td style="border: none">
+                <span>
+                  {{ verboseAnalyzerOutput.result_priority }}
                 </span>
               </td>
             </tr>
+            <tr v-if="verboseAnalyzerOutput.references !== undefined">
+              <td colspan="2" style="border: none">
+                <strong>References:</strong>
+                <ul>
+                  <li v-for="(item, index) in verboseAnalyzerOutput.references" :key="index">
+                    <a @click="contextLinkRedirect(item)">{{ item }}</a>
+                    <v-dialog v-model="redirectWarnDialog" max-width="515" :retain-focus="false">
+                      <ts-link-redirect-warning
+                        app
+                        @cancel="redirectWarnDialog = false"
+                        :context-url="contextUrl"
+                      ></ts-link-redirect-warning>
+                    </v-dialog>
+                  </li>
+                </ul>
+              </td>
+            </tr>
             <tr>
               <td style="border: none">
                 <strong>Last run:</strong>
               </td>
+              <td style="border: none">
+                <span> {{ timeline.created_at }} UTC </span>
+              </td>
+            </tr>
+            <tr>
+              <td width="80" style="border: none">
+                <strong>Status:</strong>
+              </td>
+              <td style="border: none">
+                <span>
+                  {{ verboseAnalyzerOutput.result_status }}
+                </span>
+              </td>
+            </tr>
+            <tr v-if="Object.keys(getAnalyzerOutputMetaData).length !== 0">
+              <td colspan="2" style="border: none">
+                <strong>Connected Events</strong>
```

Comment:
As discussed I would like to find some other wording here. `Events` is pretty broad and overloaded term.

Some ideas
- Supportive output
- Analyzer Output
- Result artifacts
- Explore output

Reply:
Thanks for the great suggestions. I like `Result artifacts` and will change it.